src/NuGet.Packaging/PackagePathResolver.cs

```diff
@@ -67,5 +67,10 @@ public virtual string GetInstalledPackageFilePath(PackageIdentity packageIdentit
         {
             return PackagePathHelper.GetInstalledPackageFilePath(packageIdentity, this);
         }
+
+        public virtual string GetInstalledPackageFolder(PackageIdentity packageIdentity)
```

Comment:
Perhaps, add a (verbose) comment here, stating, how this differs from GetInstalledPath